scripts/parser/analyze.js

```diff
@@ -14,13 +14,23 @@
 import { readFileSync, writeFileSync } from 'fs';
 import { getTsconfig } from 'get-tsconfig';
 import minimist from 'minimist-lite';
+import resolve from 'resolve/sync.js';
+import nodepath from 'path';
```

Comment:
GitHub eslint action:

`path` import should occur before import of `get-tsconfig`

Reply:
https://github.com/genesiscommunitysuccess/custom-elements-lsp/pull/52/commits/65fa72947e8c7efdda289f20c59119d2df106480